### DISCUSSION TITLE
Add alias support in the configuration

### DISCUSSION THREAD

**Discussion by davzucky** [DavidVujic/python-polylith#304] [Ideas]
**Is your feature request related to a problem? Please describe.**
The current version of Polylith supports aliases to check dependencies. This is great, however it can start to be hard to keep track. It would be nice to have the ability to have all the aliases you need for your project under the configuration like `[tool.polylith.alias]`

**Describe the solution you'd like**
Add a new configuration section in the `pyproject.toml` to manage aliases

**Additional context**
We want to automate the check of the poly workspaces using pre-commit. however we don't want to have to maintain the list of aliases separately 

PS: We will merge that back the pre-commit configuration after running it.

**Comment by DavidVujic** [comment]
Thank you for the suggestion! Do you need aliases? I'm thinking that maybe there is a scenario that I have missed out on when analyzing the source code (using the data from lock files) and that should be fixed. Aliases should no longer be needed (that's my goal, at least 😄 ).

**Comment by davzucky** [comment]
If you removed the need for aliases, that would be even better. Let me use the latest version, as I still use what is in the current documentation. I will come back to you. I have few libraries like that I know need aliases

**Comment by DavidVujic** [comment]
(moving this to the Discussions and the Ideas section)
